site/content/docs/5.2/components/spinners.md

```diff
@@ -205,9 +205,7 @@ Used for creating the CSS animations for our spinners. Included in `scss/_spinne
 
 
 [color]:   {{< docsref "/utilities/colors" >}}
-[display]: {{< docsref "/utilities/display" >}}
```

Comment:
Not sure if this and the next change should ring a bell and mean we are missing a link/or we need to tweak the wording in this page.